### PR TITLE
Try to fix css

### PR DESCRIPTION
**Pull request by PierreRambaud**
Try to fix css and remove absolute.

### CODE REVIEW

**Comment by marionf** [comment]
@PierreRambaud 

Every time I click on the button it goes back up and becomes smaller

https://drive.google.com/open?id=1-MbfIOK8TzGYWTDeDaNQhLNQAuFgzkmZ

capture d ecran_704

Can you remove the space between the enable button and the dropdown ?

capture d ecran_706

**Comment by PierreRambaud** [comment]
Should be ok now.

File: views/templates/admin/page.tpl [hunk 1]
```diff
@@ -110,27 +110,25 @@
...
+                        <div v-if="module.attributes.nbRates > 0" class="float-left" v-bind:class="getAvgRateClass(module.attributes.avgRate)">
                             ([[ module.attributes.nbRates ]])
                         </div>
+
+                        <div class="float-right module-price">
+                            <span v-if="module.attributes.price === 0" class="pt-2" >
+                                {l s='Free'}
+                            </span>
+                            <span v-else>
+                                [[ module.attributes.price ]] {$currency_symbol}
```
Comment (Quetzacoalt91) [inline views/templates/admin/page.tpl]: @MrBaiame FYI, with some currencies the symbol is before the amount (i.e with GBP).

**Comment by Quetzacoalt91** [comment]
Thank you @PierreRambaud

**Pull request merged**